### PR TITLE
Adding Ordering, referring and contract physician to fee sheet

### PR DESCRIPTION
**Pull request by teryhill**
Adding Ordering, referring and contract physician to fee sheet #428.

In the HCFA 1500 Print this only changes the referring provider. If a box 17 provider is supplied on the misc billing form it is used. If there is no misc billing box 17 information, the fee sheet is checked for referring then the patient data field.

They are separate fields in the electronic form so you can have an ordering and a referring.

### CODE REVIEW

File: library/globals.inc.php [hunk 1]
```diff
@@ -961,6 +961,48 @@
...
+
+    'contract_physician_in_feesheet_name' => array(
+      xl('Label Title for Contract Physician on Fee Sheet'),
+      'text',                           // data type
+      'Contractor',
+      xl('Label Title for Contract Physician on Fee Sheet')
+    ),
+    
+    'allow_appointments_in_feesheet' => array(
+      xl('Allow Schedualing Appointments from Fee Sheet'),
```
Comment (tmccormi) [inline library/globals.inc.php]: Spelling issue -  Sched - Dualing sounds dangerous

File: library/globals.inc.php [hunk 2]
```diff
@@ -961,6 +961,48 @@
...
+      'bool',                           // data type
+      '0',                              // default = false
+      xl('Allow Schedualing Appointments from the Fee Sheet.')
```
Comment (tmccormi) [inline library/globals.inc.php]: Same here

File: library/globals.inc.php (hunk 1 above)
Comment (teryhill) [inline library/globals.inc.php]: Fixed

File: library/globals.inc.php (hunk 2 above)
Comment (teryhill) [inline library/globals.inc.php]: Fixed

**Pull request merged**